packages/amplify_datastore/example/android/build.gradle

```diff
@@ -1,6 +1,7 @@
 buildscript {
     ext.kotlin_version = '1.6.10'
     repositories {
+        mavenLocal()
```

Comment:
Why are we checking in this change?

Reply:
ughhh it shouldn't be there. Will remove.